ffmpeg/ffmpeg.go

```diff
@@ -359,6 +359,15 @@ func (t *Transcoder) Transcode(input *TranscodeOptionsIn, ps []TranscodeOptions)
 			default:
 				return nil, ErrTranscoderPrf
 			}
+			if p.Profile.Framerate == 0 && p.Accel == Nvidia {
+				// When the decoded video contains non-monotonic increases in PTS (common with OBS)
```

Comment:
That's correct. If FPS passthrough is not enabled we use the `fps` filter, where we manually introduce monotonic timestamps https://github.com/livepeer/lpms/blob/0054a3bd93aa13fbf143946642ece4888a4afcce/ffmpeg/filter.c#L221-L223 (because it doesn't work well without it)